ExampleMod/Content/Items/Tools/ExampleHook.cs

```diff
@@ -117,8 +118,8 @@ internal class ExampleHookProjectile : ModProjectile
 				Color drawColor = lightColor;
 
 				//Draw chain
-				spriteBatch.Draw(chainTexture, new Vector2(center.X - Main.screenPosition.X, center.Y - Main.screenPosition.Y),
-					new Rectangle(0, 0, chainTexture.Width, chainTexture.Height), drawColor, projRotation,
+				spriteBatch.Draw(chainTexture.Value, new Vector2(center.X - Main.screenPosition.X, center.Y - Main.screenPosition.Y),
```

Comment:
Is this intended? (Is Asset from TML now or Vanilla?) I'd kind of expect draw to be able to accept an Asset.. huh..